### ISSUE TITLE
When gallery first init with single image and slides updated after, visibility of controls not updating

### ISSUE THREAD

**Issue by berkayyildiz** [sachinchoolur/lightGallery#1069]
## Description

When gallery first init with single image and images added after it not updates visibility of controls.

### Steps to reproduce

Here exact reproduction:
https://codepen.io/berkay-yildiz/pen/OJmJdXN

**Comment by sachinchoolur** [comment]
Thank you for reporting this issue.

Please use `refresh` method instead of `updateSlides` for now.

```js

const addItems = () => {
dynamicGallery.refresh([
    {
      src:
        "https://images.unsplash.com/photo-1477322524744-0eece9e79640?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1400&q=80",
      responsive:
        "https://images.unsplash.com/photo-1477322524744-0eece9e79640?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=480&q=80 480, https://images.unsplash.com/photo-1477322524744-0eece9e79640?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=80 800",
      thumb:
        "https://images.unsplash.com/photo-1477322524744-0eece9e79640?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=240&q=80"
    },
    {
      src: "//www.youtube.com/watch?v=egyIeygdS_E",
      poster: "https://img.youtube.com/vi/egyIeygdS_E/maxresdefault.jpg",
      thumb: "https://img.youtube.com/vi/egyIeygdS_E/maxresdefault.jpg"
    }
], 0)
}
```

I'll push a fix for `updateSlides` and update the documentation ASAP

**Comment by sachinchoolur** [comment]
Fixed the issue in version 2.1.7. 
But, please use `refresh()` instead of `updateSlides()` unless you want to add new slides when gallery is opened
